readthedocs/oauth/services/github.py

```diff
@@ -58,14 +55,12 @@ def sync_organizations(self):
         remote_repositories = []
 
         try:
-            orgs = self.paginate('https://api.github.com/user/orgs')
+            orgs = self.paginate("https://api.github.com/user/orgs?per_page=100")
```

Comment:
you should be able to pass this as s keyword arg, as https://github.com/readthedocs/readthedocs.org/blob/30ad791bbaeed405f19c5e41944c3a1f24b38ae6/readthedocs/oauth/services/gitlab.py#L78-L85